docs/guides/integrating-optimize.md

```diff
@@ -0,0 +1,118 @@
+
+1. On the right side of the **Home** page, select **Create New > New Report**. Here we’ll take a look at a single process, though you can also view combined processes and decision reports.
+2. Click the text box under **Select Process** and select the process you’d like to analyze. 
+3. Select the type of report you’d like to use on the left side of the **Create new Report** box. As with dashboards, Optimize offers preconfigured templates such as heatmaps and tables. We’ll begin with a heatmap.
+4. Click **Create Report**.
+5. Set up and customize your report. Begin by naming your report in the text box at the top of the page, pre-filled with **New Report**.
+6. In the gray text box to the right, confirm your data source, and select what you’d like to review from the process (in this case, we are viewing flow nodes.) You can also group by topics such as duration or start date.
+7. If you’d like, filter the process instance or flow nodes. For example, you can filter by duration, only viewing process instances running for more than seven days.
+8. Finally, you have the option to view particular sets of data from the instance, like instance count or absolute value, by selecting the gear icon to the left of your data customization. You can also choose how you’d like to visualize your data in the box beneath **Visualization** (i.e. bar chart, pie chart, etc.). Once you’ve made your selections, click **Save**.
+
+## Collections
+
+Within your collection, you can also access the **Users** and **Data Sources** tabs to further customize your collection.
+
+### Users
+
+Within the **Users** tab, review the users and user groups with access to your collection.
+
+Select **Add** to search for a user or user group to add, of which may be assigned as a viewer, editor, or manager.
+
+### Data sources
+
+Within the **Data Sources** tab, review and add source(s) of your data to create reports and dashboards inside the collection.
+
+## Additional analysis
+
+Now that we’ve created data sets within the **Home** page, let’s shift into the **Analysis** tab.
+
+Inside this tab, you’ll notice **Outlier Analysis** and **Branch Analysis**.
+
+### Outlier analysis
+
+Inside **Outlier Analysis**, we utilize heatmap displays. Click **Select Process**, choose your process, and choose your version.
+
+**Image placeholder**
+
+Within the example above, we notice increased heat (recognized as red) surrounding our invoice approved gateway. Several instances have taken significantly longer than average, so we may choose to take a closer look at these instances by downloading the instance IDs, or viewing the details for further analysis. Here, you can also find if the outliers have a shared variable.
+
+### Branch analysis
+
+Inside the **Branch Analysis** tab, we can select a ​​process and analyze how particular gateway branches impact the probability of reaching an end event.
```

Comment:
Much of Optimize's functionality is abstract. "Gateway" and "End Event" are technical terms that will go over the heads of our target audience (business users).

I recommend providing a concrete example of what it can do.
"Here, we can see that whether Lafayette travels by land or by sea has a huge effect on whether he successfully overthrows the monarchy"
(Niall has a funny demo)

Reply:
This section follows with an example outlining the likelihood of an invoice being processed once it reaches the gateway for approval -- is there a particular example Camunda has outlined you would like to showcase instead, and/or screenshots you could share? Happy to adjust.

Would love to see Niall cover an overthrow of the monarchy!